fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/RuleSet.java

```diff
@@ -350,9 +350,9 @@ public void addNestLoopJoinImplementationRule() {
 
     public void addAutoJoinImplementationRule() {
         this.implementRules.add(HashJoinImplementationRule.getInstance());
-        this.implementRules.add(MergeJoinImplementationRule.getInstance());
-        // TODO: turn it on
-        // this.implementRules.add(NestLoopJoinImplementationRule.getInstance());
+        // TODO: implement merge join
+        // this.implementRules.add(MergeJoinImplementationRule.getInstance());
```

Comment:
why comment MergeJoinImplementation?

Reply:
It has not been implemented, and this rule is never used.